docs/tutorials/best-practices/scale-coder.md

```diff
@@ -0,0 +1,309 @@
+# Scale Coder
+
+This best practice guide helps you prepare a low-scale Coder deployment that you can
+scale up to a high-scale deployment as use grows, and keep it operating smoothly with a
+high number of active users and workspaces.
+
+## Observability
+
+Observability is one of the most important aspects to a scalable Coder deployment.
+When you have visibility into performance and usage metrics, you can make informed
+decisions about what changes you should make.
+
+[Monitor your Coder deployment](../../admin/monitoring/index.md) with log output
+and metrics to identify potential bottlenecks before they negatively affect the
+end-user experience and measure the effects of modifications you make to your
+deployment.
+
+- Log output
+  - Capture log output from Loki, CloudWatch logs, and other tools on your Coder Server
+  instances and external provisioner daemons and store them in a searchable log store.
+  - Retain logs for a minimum of thirty days, ideally ninety days.
+  This allows you to look back to see when anomalous behaviors began.
+
+- Metrics
+  - Capture infrastructure metrics like CPU, memory, open files, and network I/O for all
+  Coder Server, external provisioner daemon, workspace proxy, and PostgreSQL instances.
+
+### Capture Coder server metrics with Prometheus
```

Comment:
This feels too separated from the content above about infrastructure metrics.

The idea is that you need to capture 2 kinds of metrics: infrastructure, and Coder-specific, and you want them both to be available in the same system for alerting and dashboards.

Reply:
I wonder if that's because the subheadings under **Observability** are serving different purposes, with `### Observability key metrics` providing more information about a specific aspect, and `### How to capture Coder server metrics with Prometheus` showing steps.

I rearranged things in f110af3e9c7190f4d8e51c87c80d85172958f8a2 to see if that helps the read-through, but I think that if we can add an additional method for capturing metrics and logs, we can split that out into its own H2

Does it make sense to add an example of how to stream coderd or provisionerd logs?

Reply:
I worry a bit about mixing high-level advice with specific step-by-step instructions, since the step-by-step stuff could change and people might not realize they also have to update this document. I'd much prefer linking so that step-by-step instructions appears in one place for any specific task.

Reply:
fair - that'd be a lot easier with reusable content (embeds/includes/partials, whatever), but as it is now, let's get this out and add relevant links around as we find them